applications/MappingApplication/tests/mapper_test_case.py

```diff
@@ -83,6 +83,85 @@ def setUp(self):
             condition.SetValue(KM.TEMPERATURE, default_scalar)
             condition.SetValue(KM.VELOCITY, default_vector)
 
+class BeamMapperTestCase(KratosUnittest.TestCase):
```

Comment:
if its only used for the beam-mapper, then I suggest to move it to `test_beam_mapper.py`